ARCs/arc-draft_interface-detection.md

```diff
@@ -0,0 +1,86 @@
+---
+arc: <to be assigned>
```

Comment:
You can use the ARC-72, I will close the other PR

Reply:
Well, this PR has two ARCs.  Maybe I should have separated them, but I wanted to submit them at the same time because they are to some degree related.  Should I use ARC-72 for interface detection and something else for the NFT spec?

Reply:
In that case, you can use 72 for NFT-ARC & 73 for interface detection